hotelReservation/config.json

```diff
@@ -18,5 +18,5 @@
   "SearchPort": "8082",
   "UserPort": "8086",
   "UserMongoAddress": "mongodb-user:27017",
-  "KnativeDomainName":  "default.10.108.189.25.sslip.io:80"
+  "KnativeDomainName": "default.192.168.1.240.sslip.io:80"
```

Comment:
I think we've already set the default KnativeDomainName as empty. Otherwise, it will block the default docker-compose environment from running correctly, also I believe different Knative deployment needs to change the default value, right?